R/clustering.R

```diff
@@ -720,17 +714,12 @@ RunLeiden <- function(
       },
     "igraph" = {
       input <- if (inherits(x = object, what = 'list')) {
-        if (is.null(x = weights)) {
```

Comment:
```suggestion 
     if(!is.null(weights)) {
          char <- as.character(deparse(sys.call()))
          .Deprecated(new = gsub(paste(",", y), "", char),
                     package="Suerat", 
                     old = char)
        }
```

Deprecate `weights` input (give warning that will be removed in future and should not be called in scripts).